libs/ui/src/reports/report_generated_metadata.tsx

```diff
@@ -7,7 +7,7 @@ import {
 } from '@votingworks/types';
 import { LabeledValue, ReportMetadata } from './report_header';
 
-export function AdminReportMetadata({
+export function ReportGeneratedMetadata({
```

Comment:
The component had `Admin` in the name, but nothing `Admin` specific. Renaming to reuse as the header for the Ballots Printed Report